viscy/light/data.py

```diff
@@ -344,13 +351,14 @@ def __init__(
         augment: bool = True,
         caching: bool = False,
         normalize_source: bool = False,
-        ground_truth_masks: str = None,
+        ground_truth_masks: Optional[Path] = None,
         train_z_scale_range: tuple[float, float] = [0, 0],
         train_noise_std: float = 0.0,
         train_patches_per_stack: int = 1,
+        predict_scale_source: Optional[float] = None,
```

Comment:
Here how do we select which source channel gets the scaling or does it apply it to both?

Reply:
Never mind we set on the config file which channels are source and target. Thanks